.github/workflows/bridge-ui.yml

```diff
@@ -0,0 +1,36 @@
+name: Bridge UI CI/CD
+
+on:
+  push:
+    branches-ignore:
+      - release-please-*
+    paths:
+        - "packages/bridge-ui/**"
+
+jobs:
+  deploy-bridge-ui-preview: 
```

Comment:
for all deployment, I hope to use a patten like this:

deploy_[name]_[mainnet|hekla|...]_[preview|production\